packages/react-components/src/components/CameraButton.test.tsx

```diff
@@ -25,9 +25,9 @@ describe('CameraButton strings should be localizable and overridable', () => {
       cameraButton: { offLabel: Math.random().toString(), onLabel: Math.random().toString() }
     });
     const component = mountWithLocalization(<CameraButton showLabel={true} />, testLocale);
-    expect(component.text()).toBe(testLocale.strings.cameraButton.offLabel);
+    expect(component.find('button').text()).toBe(testLocale.strings.cameraButton.offLabel);

```

Comment:
If you are curious why these unit tests needed updated, Fluent (for A11y reasons) now have the tooltip text rendered in the element but hidden when not hovered:
![image](https://user-images.githubusercontent.com/2684369/196317622-3a049b94-c623-4ba5-b4d6-85a5193e1745.png)

This caused the enzyme `.text()` to pick up the hidden content, so instead we only grab the text from the button element (given this is what we are testing anyway):
![image](https://user-images.githubusercontent.com/2684369/196317873-1b998b5c-bc23-4e64-9a14-c3745f99aa47.png)

Reply:
Maybe this is a change to also be called out in the web UI channel so people know going forward this is needed for tests?